WordPress/Classes/ViewRelated/Me/SettingsCommon.swift

```diff
@@ -4,6 +4,14 @@ protocol SettingsController: ImmuTableController {}
 
 // MARK: - Actions
 extension SettingsController {
+    func insideNavigationController(generator: ImmuTableRowControllerGenerator) -> ImmuTableRowControllerGenerator {
+        return { row in
+            let controller = generator(row)
+            let navigation = UINavigationController(rootViewController: controller)
```

Comment:
Good catch, I was only looking at the original commit and not the PR, so I missed it.